src/components/EmergencyShutoffCard/index.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { func, object } from 'prop-types';
+import { makeStyles } from '@material-ui/styles';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import CardContent from '@material-ui/core/CardContent';
+import CardActions from '@material-ui/core/CardActions';
+import Typography from '@material-ui/core/Typography';
+import AlertIcon from 'mdi-react/AlertIcon';
+import Button from '../Button';
+import SignoffSummary from '../SignoffSummary';
+import { withUser } from '../../utils/AuthContext';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    border: `2px solid ${theme.palette.warning.dark}`,
+  },
+  space: {
+    paddingTop: theme.spacing(2),
+  },
+  cardHeader: {
+    backgroundColor: theme.palette.warning.main,
+  },
+  cardHeaderAvatar: {
+    display: 'flex',
+  },
+  cardContentRoot: {
+    padding: theme.spacing(1),
+  },
+  cardActions: {
+    justifyContent: 'flex-end',
+  },
+}));
+
+function EmergencyShutoffCard({
+  emergencyShutoff,
+  onEnableUpdates,
+  onCancelEnable,
+  onSignoff,
+  onRevoke,
+  user,
+  onAuthorize,
+  onUnauthorize,
+  ...props
+}) {
+  const classes = useStyles();
+  const requiresSignoff =
+    emergencyShutoff.scheduledChange &&
+    Object.keys(emergencyShutoff.scheduledChange.required_signoffs).length > 0;
+
+  return (
+    <Card classes={{ root: classes.root }} spacing={4} {...props}>
+      <CardHeader
+        classes={{ avatar: classes.cardHeaderAvatar }}
+        className={classes.cardHeader}
+        avatar={<AlertIcon />}
+        title={
+          <Typography component="h2" variant="h6">
+            {emergencyShutoff.scheduledChange
+              ? 'Updates are scheduled to be enabled for this product and channel'
+              : 'Updates are currently disabled for this product and channel'}
```

Comment:
Instead of `for this product and channel`, it would be great if we could substitute the actual names there.